learning_orchestra_client/utils/file_caster.py

```diff
@@ -0,0 +1,52 @@
+from os import PathLike
+
+
+def python_file_caster(source: str, filename: str = '<string>',
+                       flags: int = 0,
+                       dont_inherit: bool = False,
+                       optimize: int = -1) -> str:
+    """
+    description: Convert source into a string to later be executed by an
```

Comment:
After the PR accepted, you need generate a new doc of this project with pdoc lib and put the doc on /docs folder in gh-pages branch.